setup.py

```diff
@@ -62,7 +62,7 @@
 
 EXTRAS_REQUIRE = {
     "nncf": ["nncf>=2.11.0"],
-    "openvino": ["nncf>=2.11.0", "openvino==2024.4.1.dev20240926", "openvino-tokenizers==2024.4.1.0.dev20240926"],
+    "openvino": ["nncf>=2.11.0", "openvino==2024.5.0", "openvino-tokenizers==2024.5.0"],
```

Comment:
this version is not availbe on pypi yet, only on openvino storage.
I suppose for testing it will be better to install openvino and openvino-tokenizers explicitly in GHA using this command:
https://github.com/huggingface/optimum-intel/blob/main/.github/workflows/test_openvino_slow.yml#L53
```
pip install -U --pre --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/pre-release openvino openvino-tokenizers
```

Reply:
@eaidova please see #734